examples/neutrion/convert_trtc_v4_to_hf.py

```
@@ -0,0 +1,245 @@
+    emit("model.norm.weight", torch.from_numpy(final_norm.copy()).to(torch.bfloat16))
+    embed_out, off = _int8_record(mm, off)
+    assert (embed_out[0], embed_out[1]) == (vocab, hidden)
+    emit("lm_head.weight", _dense_int8(embed_out))
+    if embed_in is None:  # tied: the lm_head record serves both
+        emit("model.embed_tokens.weight", _dense_int8(embed_out))
+    assert off == mm.size, f"reader ended at {off}, file is {mm.size}"
+    shards.append(shard_tensors)
+
+    names = [f"model-{i + 1:05d}-of-{len(shards):05d}.safetensors" for i in range(len(shards))]
+    total = 0
+    for name, tensors in zip(names, shards):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)